File: Sort/bubbleSort.py
def bubblesort(a):
    n=len(a) 
    for j in range(n-1,0,-1):
        for i in range(j):
            if a[i]>a[i+1]:
                temp=a[i] 
                a[i]=a[i+1]
                a[i+1]=temp

A = [3, 5, 8, 9, 6, 2]
print('Original Array:',A)
bubblesort(A)
print('Sorted Array:',A)

File: Sort/bubbleSort.go
package main

import "fmt"

func bubbleSort(items []int) {
	n := len(items)
	for j := n - 1; j > 0; j-- {
		for i := 0; i < j; i++ {
			if items[i] > items[i+1] {
				temp := items[i]
				items[i] = items[i+1]
				items[i+1] = temp
			}
		}
	}

}

func main() {
	items := []int{1, 2, 6, 8, 5, 4, 3, 7}
	fmt.Println(items)
	bubbleSort(items)
	fmt.Println(items)

}
